lib/Mojolicious/Plugin/AssetPack/Preprocessor/Scss.pm

```diff
@@ -188,8 +188,8 @@ sub _import_path {
   my ($ext, $name, $path) = (pop @rel, pop @rel);
 
   for my $p (map { File::Spec->catdir($_, @rel) } @$include_paths) {
-    return $path if -r ($path = catfile $p, "$name.$ext");
-    return $path if -r ($path = catfile $p, "_$name.$ext");
+    -r ($path = catfile $p, $_) and return $path
+      for "$name.$ext", "_$name.$ext", "$name", "_$name";
```

Comment:
This is very difficult to read imo. Also, are you sure this is the right order? I would suggest something like this:

    for my $path ($name, "_$name", "$name.$ext", "_$name.$ext") {
      return $path if -f $path and -r _;
    }